src/language-js/options.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
     category: CATEGORY_JAVASCRIPT,
     type: "boolean",
     default: false,
-    description: "Put > on the last line instead of at a new line.",
+    description:
```

Comment:
Is it possible to use `deprecated` option? https://github.com/ikatyang/vnopts#deprecated

So maybe we can warn about `--jsx-bracket-same-line` on CLI `node bin/prettier index.js --jsx-bracket-same-line -l`